addons/binding/org.openhab.binding.hyperion/src/main/java/org/openhab/binding/hyperion/internal/protocol/HyperionConnection.java

```diff
@@ -150,4 +151,16 @@ private void notifyListeners(String property, Object oldValue, Object newValue)
             listener.stateChanged(property, oldValue, newValue);
         }
     }
+
+    public void connect() {
```

Comment:
I guess this is preparation for connecting/disconnecting only once with the server? Do you plan to implement this with this PR?